### PR TITLE
Fixed some issues

### PR DESCRIPTION
**Pull request by dominikl**
# What this PR does

Fixes https://github.com/openmicroscopy/openmicroscopy/issues/5038

# Testing this PR

Code review.

### CODE REVIEW

File: components/server/src/ome/formats/OMEROMetadataStore.java [hunk 1]
```diff
@@ -2203,7 +2203,6 @@ private LogicalChannel getUniqueLogicalChannel(
     			&& lc.getLightSourceSettings() == lc2.getLightSourceSettings()
     			&& compare(lc.getName(), lc2.getName())
     			&& compare(lc.getNdFilter(), lc2.getNdFilter())
-    			&& lc.getOtf() == lc.getOtf()
```
Comment (mtbc) [inline components/server/src/ome/formats/OMEROMetadataStore.java:2206]: should this be `lc2` on the right?

File: components/insight/SRC/org/openmicroscopy/shoola/env/config/OMEROInfo.java [hunk 2]
```diff
@@ -105,8 +105,7 @@ public OMEROInfo(String port, String portSSL, String hostName,
         if (encrypted == null) this.encrypted = false; 
 		else {
 			encrypted = encrypted.toLowerCase();
-			if (AgentInfo.TRUE.equals(encrypted) ||
-				AgentInfo.TRUE.equals(encrypted))
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/config/OMEROInfo.java:109]: should this be `AgentInfo.TRUE_SHORT` given the FALSE code below?

File: components/insight/SRC/org/openmicroscopy/shoola/env/config/OMEROEntry.java [hunk 3]
```diff
@@ -109,7 +109,7 @@ private static boolean getConfigAttribute(Node tag)
 		}
 		if (value == null) return true;
 		value = value.toLowerCase();
-		if (AgentInfo.TRUE.equals(value) || AgentInfo.TRUE.equals(value))
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/config/OMEROEntry.java:112]: `TRUE_SHORT`?

**Comment by dominikl** [comment]
Thanks @mtbc , you're right, your suggestions are more reasonable. Added with last commit.

File: components/blitz/src/ome/formats/OMEROMetadataStoreClient.java [hunk 4]
```diff
@@ -1719,8 +1719,7 @@ public int getDefaultGroupLevel() throws ServerError {
             else groupLevel = ImportEvent.GROUP_COLLAB_READ;
         }
         else if (perm.isWorldRead()) {
-            if (perm.isWorldWrite())  groupLevel = ImportEvent.GROUP_PUBLIC;
-            else groupLevel = ImportEvent.GROUP_PUBLIC;
+            groupLevel = ImportEvent.GROUP_PUBLIC;
```
Comment (mtbc) [inline components/blitz/src/ome/formats/OMEROMetadataStoreClient.java:1722]: yes, I suppose we don't have anything further in the enum for world-writable groups

File: components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/LocationDialog.java [hunk 5]
```diff
@@ -894,8 +894,7 @@ private void setMinimumSize()
 	private void close()
 	{
 	    int plugin = ImporterAgent.runAsPlugin();
-        if (plugin == LookupNames.IMAGE_J_IMPORT ||
-                plugin == LookupNames.IMAGE_J_IMPORT) {
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/LocationDialog.java:898]: could this perhaps be meant to be `LookupNames.IMAGE_J`?

**Comment by mtbc** [comment]
Thank you. :+1: **Good to merge** after checked by tomorrow morning's CI.

**Comment by jburel** [comment]
everything looks good today in CI

**Pull request merged**